adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala

```diff
@@ -92,6 +94,16 @@ case class FragmentRDD(rdd: RDD[Fragment],
   }
 
   /**
+   * Marks reads as possible fragment duplicates.
+   *
+   * @return A new RDD where reads have the duplicate read flag set. Duplicate
+   *   reads are NOT filtered out.
+   */
+  def markDuplicates(): FragmentRDD = MarkDuplicatesInDriver.time {
```

Comment:
is this saying that `MarkDuplicates` runs solely on the driver, or that the timer is?